File: app/src/main/java/com/orangepenguin/boilerplate/screens/userdetails/UserDetailsView.java
package com.orangepenguin.boilerplate.screens.userdetails;

import com.orangepenguin.boilerplate.BaseViewInterface;

public interface UserDetailsView extends BaseViewInterface {
    void setUsername(String username);
    void setCompany(String company);
    void setName(String name);
    void setAvatarUrl(String avatarUrl);
}

File: app/src/main/java/com/orangepenguin/boilerplate/screens/userdetails/UserDetailsView.kt
package com.orangepenguin.boilerplate.screens.userdetails

import com.orangepenguin.boilerplate.BaseViewInterface

interface UserDetailsView : BaseViewInterface {
    fun setUsername(username: String)
    fun setCompany(company: String)
    fun setName(name: String)
    fun setAvatarUrl(avatarUrl: String)
}
